include/onnxruntime/core/common/gpu_profiler_common.h

```diff
@@ -265,6 +282,10 @@ class GPUTracerManager {
     events_pending_client_mapping_.erase(pending_it);
   }
 
+  uint64_t NormalizeGPUTimestampNanoseconds(uint64_t gpu_timestamp_in_nanoseconds) {
```

Comment:
I would call it as `ConvertToCPUTimeEpoch`.

Reply:
Renamed to `NormalizeGPUTimestampToCPUEpoch`.